src/MerQure.RbMQ/Config/RabbitMqConnectionConfigurationElement.cs

```diff
@@ -1,13 +1,8 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Configuration;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace MerQure.RbMQ.Config
 {
-    public class RabbitMqConnectionConfigurationElement : ConfigurationElement
+    public class RabbitMqConnectionConfigurationElement  
     {
         public const string DefaultConnectionString = "amqp://guest:guest@localhost:5672/";
         private const string ConnectionStringPropertyName = "connectionString";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'ConnectionStringPropertyName'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1144)